deploy/openshift/openshift/99_master_multus_config_link.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 10-master-multus-config-link
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+      - name: multus-config-link.service
+        enabled: true
+        contents: |
+          [Unit]
+          Description=Link multus config to CNI config file
+          After=syslog.target
+          AssertPathExists=/etc/contrail/link_multus_config.sh
+
+          [Service]
+          Type=simple
```

Comment:
I think this needs to be a oneshot, so that the script is not executed in a loop.